docker/docker-compose-with-ccd.yml

```diff
@@ -349,19 +349,19 @@ services:
      - 8081
 
   pdf-service-api:
-    image: docker.artifactory.reform.hmcts.net/reform/pdf-service-api:f7969638ccd7c83e6ce70330fa3a16e8a6ab53ae
+    image: hmcts.azurecr.io/hmcts/cmc-pdf-service:latest
```

Comment:
I think the switch to cmc-pdf-service is ok which is what the docker for probate-back-office uses. However in the past the pdf service code didn't like this version, can you check for me the pdf links in check your anwsers, coversheet etc work in the PA journey, thanks.